service/src/main/java/com/commercetools/pspadapter/payone/mapping/BankTransferWithoutIbanBicRequestFactory.java

```diff
@@ -50,7 +50,12 @@ public BankTransferRequest createRequestInternal(@Nonnull final RequestType requ
         mapFormPaymentWithCartLike(request, paymentWithCartLike);
 
         //Despite declared as optional in PayOne Server API documentation. the Bankcountry is required
-        request.setBankcountry(request.getCountry());
+        final String bankCountry = ctPayment.getCustom().getFieldAsString(CustomFieldKeys.BANK_COUNTRY);
+        if (bankCountry == null) {
+            request.setBankcountry(request.getCountry());
```

Comment:
in case of bancontact, there is a probability that we are inserting the country value something other than `BE`. This makes a room from payone integration project to create wrong requests to payone. Should we instead make the custom-type definition to be compulsory for bancontact and iDeal? so, we don't require to check validation at this point. wdyt?

Reply:
ohh, okay, we already marked the custom type definition as mandatory. So, this makes sense now 👍

Reply:
Do you mean to put instead of type "String" you would put some other type for country attribute in the product type? Is there a type "country"?